src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class Fruit {
+    private String name;
+
+    public Fruit(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
```suggestion
    public boolean equals(Object o) {
```
use more readable name